pkg/venafi/tpp/connector_test.go

```diff
@@ -28,13 +28,15 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/http/httptest"
 	"net/url"
 	"os"
 	"path/filepath"
 	"reflect"
 	"regexp"
 	"strconv"
 	"strings"
+	"sync/atomic"
```

Comment:
This need to be added in go dependencies. 
Jenkisn build is failing

`# github.com/Venafi/vcert/v4/pkg/venafi/tpp [github.com/Venafi/vcert/v4/pkg/venafi/tpp.test]

pkg/venafi/tpp/connector_test.go:579:116: undefined: atomic.Bool

pkg/venafi/tpp/connector_test.go:580:23: undefined: atomic.Bool

FAIL	github.com/Venafi/vcert/v4/pkg/venafi/tpp [build failed]

FAIL`

Reply:
Hmm... `sync/atomic` comes from the Go standard library, which means it doesn't need to be referenced in `go.mod`. 

I will remove the use of `atomic.Bool`.